controller/common/output.go

```diff
@@ -244,6 +245,71 @@ func NewWebHook(url, target string) *Webhook {
 	return w
 }
 
+func getFieldStringValue(elog interface{}, fieldName string) (string, error) {
+	var value string
+	var ok bool
+
+	field, err := pointerstructure.Get(elog, fieldName)
+	if err != nil {
+		return value, fmt.Errorf("invalid fieldName: %s", fieldName)
+	}
+	value, ok = field.(string)
+
+	if ok {
+		return value, nil
+	} else {
+		return value, fmt.Errorf("fieldName: %s is not string", fieldName)
+	}
+}
+
+func getDetailInfoFromLog(elog interface{}) string {
+	var builder strings.Builder
+	fieldOrder := []string{
+		"/LogCommon/ClusterName",
+		"/WorkloadDomain",
+		"/WorkloadImage",
+		"/ClientWLDomain",
+		"/ClientDomain",
+		"/ClientWLImage",
+		"/ClientImage",
+		"/ServerWLDomain",
+		"/ServerDomain",
+		"/ServerWLImage",
+		"/ServerImage",
+		"/LogCommon/HostName",
+		"/LogCommon/ReportedAt",
+	}
+
+	fieldNames := map[string]string{
+		"/LogCommon/ClusterName": "cluster",
+		"/WorkloadDomain":        "namespace",
+		"/WorkloadImage":         "image",
+		"/ClientWLDomain":        "client namespace",
+		"/ClientDomain":          "client namespace",
+		"/ClientWLImage":         "client image",
+		"/ClientImage":           "client image",
+		"/ServerWLDomain":        "server namespace",
+		"/ServerDomain":          "server namespace",
+		"/ServerWLImage":         "server image",
+		"/ServerImage":           "server image",
+		"/LogCommon/HostName":    "node",
+		"/LogCommon/ReportedAt":  "time",
+	}
+
+	for _, fieldName := range fieldOrder {
+		value, err := getFieldStringValue(elog, fieldName)
+		if err == nil {
+			builder.WriteString(fmt.Sprintf("%s: %s, ", fieldNames[fieldName], value))
+		}
+	}
+
+	details := builder.String()
+	if len(details) > 0 {
```

Comment:
logically, it should check len(details) >= 2 to avoid possible(even rarely) panic in the next line